src/prefect/engine/executors/base.py

```diff
@@ -17,6 +17,9 @@ class Executor:
     def __init__(self) -> None:
         self.executor_id = type(self).__name__ + ": " + str(uuid.uuid4())
 
+    def __repr__(self) -> str:
+        return "<{}: {}>".format(type(self).__name__, type(self).__name__)
```

Comment:
Could you update this one like you did for `ResultHandler`s?  So `<Executor: {}>.format(...)`

Reply:
Yep, just forgot to add it with the previous commit.